cmd/tink-cli/Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM alpine:3.11
-
-CMD sleep infinity
+FROM golang:1.15-alpine
+RUN apk add --no-cache --update --upgrade ca-certificates make git
+COPY . /usr/myapp
+WORKDIR /usr/myapp
+RUN make cli
 
+FROM alpine:3.11
 RUN apk add --no-cache --update --upgrade ca-certificates
-
-COPY tink-cli /bin/tink
-COPY sample.tmpl /tmp
+COPY --from=0 /usr/myapp/cmd/tink-cli/tink-cli /usr/bin/tink-cli
+ENTRYPOINT ["/usr/bin/tink-cli"]
```

Comment:
ditto, lets do ENTRYPOINT, RUN, COPY to make best use of cache and keep groupings of metadata vs content